demos/sahi/requirements.txt

```diff
@@ -0,0 +1,3 @@
+opencv-python==4.2.0.32
+sahi==0.10.5
+yolov5==6.1.8
```

Comment:
What about yolov5==6.2.0? Have you tried the latest version? Are there any issues? If not, we may use the latest :)